presto-main/src/main/java/io/prestosql/connector/informationschema/InformationSchemaMetadata.java

```diff
@@ -288,7 +288,7 @@ public ConnectorTableProperties getTableProperties(ConnectorSession session, Con
         }
```

Comment:
improve a commit message:

```
Search table prefixes for "columns" table only
```

also please explain it is enough to search for table prefixes for `columns` table only